README.md

```diff
@@ -21,12 +21,12 @@ Git tags are used to mark minor and patch release versions.
 ## Reading the latest version of the documentation
 
 The latest version of the P4Runtime v1 specification is available:
-* [here](https://p4.org/p4runtime/spec/master/P4Runtime-Spec.html) in **HTML**
+* [here](https://p4.org/p4runtime/spec/main/P4Runtime-Spec.html) in **HTML**
   format
-* [here](https://p4.org/p4runtime/spec/master/P4Runtime-Spec.pdf) in **PDF**
+* [here](https://p4.org/p4runtime/spec/main/P4Runtime-Spec.pdf) in **PDF**
```

Comment:
These links are referenced in p4.org, but since you mention there is automatic redirection, it should not be an issue